FFXIVClientStructs/FFXIV/Component/Text/TextModule.cs

```diff
@@ -23,6 +23,10 @@ public unsafe partial struct TextModule {
     [VirtualFunction(9)]
     public partial Utf8String* ProcessMacroCode(Utf8String* output, byte* input);
 
-    [VirtualFunction(16)] //args is a pointer to a deque with the args, if any
-    public partial bool FormatString(byte* input, void* args, Utf8String* output);
+    [Obsolete("Use FormatString with StdDeque<TextParameter>*")]
+    public bool FormatString(byte* input, void* localParameters, Utf8String* output)
```

Comment:
you can remove this overload instead of obsoleting it, that's fine here

Reply:
If you say so. ![Okayge](https://cdn.frankerfacez.com/emoticon/410314/1)